modules/orchestrator/services/runtime/runtime.go

```diff
@@ -1698,11 +1704,11 @@ func (r *Runtime) FullGC() {
 			break
 		}
 		for i := range runtimes {
-			if runtimes[i].Workspace == string(apistructs.ProdWorkspace) {
-				r.fullGCForSingleRuntime(runtimes[i].ID, keep)
-			} else {
-				r.fullGCForSingleRuntime(runtimes[i].ID, 1)
+			keep, ok := rollbackCfg[runtimes[i].ProjectID][strings.ToUpper(runtimes[i].Workspace)]
+			if !ok || keep < 0 || keep > 100 {
```

Comment:
keep <= 0 is correct

```suggestion
			if !ok || keep <= 0 || keep > 100 {
```

Reply:
done